### ISSUE TITLE
Post Name with Comma breaks CSV Download Export

### ISSUE THREAD

**Issue by marxphil** [podlove/podlove-publisher#1231]
### Expected behavior
If ``$post->post_title`` has commas in it's value, it won't be intepreted as multiple values/columns.

### Actual behavior
``$post->post_title`` isn't escaped, so that a title with commas (i.e. ``my diary, day 233``) is interpreted as two columns. This leads to wrong/shifted values in the columns.

### Changes needed / Solutions
Add following code to /lib/downloads_list_data.php before #L38
```php

<?php
if(stristr($post->post_title, ",") !== false) {
$post->post_title='"'.addslashes($post->post_title).'"';
}
```

**Comment by eteubert** [comment]
Export works for me. If I have a title including a comma, it's automatically wrapped in quotes during the csv export.

**Comment by marxphil** [comment]
Thanks for checking.

Opening the file in a text editor shows the quotes.
I've opened the file in Microsoft Excel first and the quotes will be stripped then.

If someone needs a workaround:
To open the file in Excel, don't double-click the csv file or try to open it with the "Open file" dialog.
Instead, change the file extention from .csv to .txt and open the file in Excel using the "Open file" dialog. The Import Assistant starts and respects the quotes. 
Caution: Drag'n'dropping the file into Excel skips the Import Assistant and the quotes will be stripped again. You explicity need to use the "Open file" dialog.

Tested on Excel 365.

Case closed, sorry for trouble.
